Sources/General/Array.c

```diff
@@ -259,6 +259,7 @@ void PointArray2D_Destruct(PointArray2D *me)
 {
     for (int row = 0; row < me->size; row++)
     {
+        PointArray1D_Destruct(me->data[row]);
```

Comment:
Presumably me->data[row] might be NULL if PointArray2D_ConstructRow has not been called for that row?